### PR TITLE
Flyout Navigation improvements (Issue/2928)

### PR DESCRIPTION
**Pull request by henrijoss**
### Pull Request Checklist

- [x] I have read and accept the [Contributor License Agreement](https://opensource.porsche.com/docs/cla)

#### References

- Preview: https://designsystem.porsche.com/issue/2928

#### Scope

Short description of the pull request scope

#### Resolved Issue

Resolves [#2928](https://github.com/porsche-design-system/porsche-design-system/issues/2928)

### CODE REVIEW

File: packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx [hunk 1]
```diff
@@ -92,6 +91,11 @@ export class FlyoutNavigation {
     setScrollLock(false);
   }
 
+  public componentDidRender(): void {
+    // showModal needs to be called after render cycle to prepare visibility states of dialog in order to focus the dismiss button correctly
```
Comment (denyo) [inline packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx:95]: Bad copy & paste with modal references?

File: packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx [hunk 2]
```diff
@@ -153,9 +157,11 @@ export class FlyoutNavigation {
   }
 
   private setDialogVisibility(isOpen: boolean): void {
-    if (isOpen === true) {
+    // TODO: SupportsNativeDialog check
+    // Only call showModal/close on dialog when state changes
```
Comment (denyo) [inline packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx:161]: Here, too.

File: packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx (hunk 1 above)
Comment (henrijoss) [reply packages/components/src/components/flyout-navigation/flyout-navigation/flyout-navigation.tsx:95]: showModal is the name of the function of the native dialog

File: packages/components/CHANGELOG.md [hunk 3]
```diff
@@ -26,12 +26,17 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 
 #### Changed
 
+- `Flyout Navigation`: Improved validation & Active identifier isn't automatically updated anymore
```
Comment (stephanschroeter) [inline packages/components/CHANGELOG.md:29]: Either write active identifier as prop or attribute syntax

**Review by stephanschroeter — APPROVED** [review]

**Pull request merged**